bin/elixir

```diff
@@ -206,8 +206,8 @@ SCRIPT_PATH=$(dirname "$SELF")
 if [ "$OSTYPE" = "cygwin" ]; then SCRIPT_PATH=$(cygpath -m "$SCRIPT_PATH"); fi
 if [ "$MODE" != "iex" ]; then ERL="-noshell -s elixir start_cli $ERL"; fi
 
-if [ "$OS" != "Windows_NT" ]; then
-  if test -t 1 -a -t 2; then ERL="-elixir ansi_enabled true $ERL"; fi
+if [ "$OS" != "Windows_NT" -a -t 1 -a -t 2 -a ! -z "$NO_COLOR"]; then
```

Comment:
As mentioned in the now deleted commented, `-t` is not going to work as is here. Also please use && instead.

Reply:
`-t fd` is POSIX standard, so if it is ran on POSIX compliant OS then it should work, otherwise this is an OS bug.

Reply:
You are correct. It seems the issue is not a matter of standard, but rather people not following the standard. Please see the commit that introduced it:

https://github.com/elixir-lang/elixir/commit/a2662a4b511e52c4dc44ad4acfafd44ac5861d05

If you have a Windows machine and you can verify if it works on MinGW and Cygwin, then we will gladly accept the change, otherwise I would prefer to leave it as is (especially we want to avoid using `-a`, so the number of test calls end up being the same).

Reply:
With change to use `&&` it shouldn't be a problem, as `&&` lazy evaluates.

Reply:
IIRC, the issue was that it would not even parse it. Again, I will be glad to change it if someone actually verifies it on MinGW and Cygwin.